test/test-clone.sh

```diff
@@ -144,12 +144,14 @@ begin_test "clone ClientCert"
   set -e
   reponame="test-cloneClientCert"
   setup_remote_repo "$reponame"
+  pwd
   out="$(clone_repo_clientcert "$reponame" "$reponame")"
```

Comment:
I can't use `tee` exactly, since it swallows errors. If `clone_repo_clientcert()` returns an error, I want the test to halt.